plugins/virtualenvwrapper/virtualenvwrapper.plugin.zsh

```diff
@@ -1,48 +1,30 @@
-virtualenvwrapper='virtualenvwrapper.sh'
-virtualenvwrapper_lazy='virtualenvwrapper_lazy.sh'
+function {
+    setopt nullglob
```

Comment:
I suggest also set `local_options` here, so that it won't change user's shell options.

Alternatively, don't set `nullglob` here. Your code does not seem to use this feature.

Reply:
Good point, will remove it.